Sources/Latte+UniversalLink.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+public protocol LatteLinkHandler {
+    func handle(
+        context: UINavigationController,
+        authgear: Authgear,
+        handler: @escaping Latte.ResultHandler<Void>
+    )
```

Comment:
The completion handler of universal link should be a plain `(Result<Void, Error>) -> Void`, since some universal links does not require UI (e.g. login link).

I think we should dismiss the handle before signaling completion.